### DISCUSSION TITLE
[New LXC Request] TriliumNext Notes LXC

### DISCUSSION THREAD

**Discussion by ghost** [tteck/Proxmox#3524] [Ideas]
Saw that Trilium was going into maintenance mode, but TriliumNext Notes is a community oriented fork. I felt like it would make sense to add. 

Reddit Post
https://www.reddit.com/r/selfhosted/comments/1els6ui/triliumnexts_first_stable_release_is_now_available/
GitHub Page
https://github.com/TriliumNext/Notes/releases/tag/v0.90.3

Server Install is the same

**Comment by tteck** [comment]
You're late, https://github.com/tteck/Proxmox/commit/e428bb8604b08d7cef403e6dc07a6662ba2598cf
